src/components/Button.module.css

```diff
@@ -0,0 +1,20 @@
+button {
+  cursor: pointer;
+  border: none;
+  background-color: var(--cyan);
+  font-size: 1.5rem;
+  color: white;
+}
+
+.medium {
+  height: 3.5rem;
+  width: 13rem;
+}
+
+.big_radius {
+  border-radius: 2rem;
+}
+
+.getStarted:hover {
```

Comment:
What is semantic meaning here for getStarted? And why it only has a hover style? Other variants don't need any hover statuses?